packages/bpk-component-navigation-bar/src/BpkNavigationBarButtonLink.tsx

```diff
@@ -20,14 +20,9 @@ import type { ComponentProps, MouseEvent, ReactNode } from 'react';
 
 // @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
 import { BpkButtonLink } from '../../bpk-component-link';
-import { cssModules } from '../../bpk-react-utils';
 
 import { BAR_STYLES, type BarStyle } from './BpkNavigationBar';
 
-import STYLES from './BpkNavigationBarButtonLink.module.scss';
```

Comment:
We can rely on the Style for BpkButtonLink now. We don't need to replicate it here.